### ISSUE TITLE
BiSeNetV2模型进行pridict返回的是tuple类型的数据，报错。

### ISSUE THREAD

**Issue by FelixJiao** [Deeachain/Segmentation-Pytorch#3]
![image](https://user-images.githubusercontent.com/33050328/98438192-a1f9b280-2123-11eb-88b9-91105b6be6d0.png)

**Comment by Deeachain** [comment]
我猜测你使用的`predict.py`，这里是我代码没有完善好的问题。由于BisenetV2模型输出有5个值：`logits, logits_aux2, logits_aux3, logits_aux4, logits_aux5_4`，我们只需要第一个值。需要将你的预测代码稍作修改，在`predict.py`中的32行`output = model(input_var)`后面添加一个判断即可解决问题，代码已经更新上传，相似问题之前在`predict_sliding.py`中的39-70行已经解决，由于我没有使用`predict.py`，所以在之前的修改中漏更新了。
```
if type(output) is tuple:
    output = output[0]
```

**Comment by Deeachain** [comment]
> 我猜测你使用的`predict.py`，这里是我代码没有完善好的问题。由于BisenetV2模型输出有5个值：`logits, logits_aux2, logits_aux3, logits_aux4, logits_aux5_4`，我们只需要第一个值。需要将你的预测代码稍作修改，在`predict.py`中的32行`output = model(input_var)`后面添加一个判断即可解决问题，代码已经更新上传，相似问题之前在`predict_sliding.py`中的39-70行已经解决，由于我没有使用`predict.py`，所以在之前的修改中漏更新了。
> 
> ```
> if type(output) is tuple:
>     output = output[0]
> ```

我没有测试是否成功，如果还是失败希望可以反馈给我

**Comment by FelixJiao** [comment]
可以解决了，多谢。
